_includes/release-notes/overview.md

```diff
@@ -1,5 +1,5 @@
 
-The Release Notes provide high-level coverage of the improvements and additions that have been implemented in TinyMCE 5.0 and document known problems in this release, as well as notable bug fixes, deprecated functionality, and other details.
+{{site.productname}} 5.0 was released for {{site.enterpriseversion}} and {{site.cloudname}} on Monday, February 18<sup>th</sup>, 2019. It includes {{site.productname}} 5.0.0 and additional changes to premium plugins. The Release Notes provide high-level coverage of the improvements and additions that have been implemented in {{site.productname}} 5.0 and document known problems in this release, as well as notable bug fixes, deprecated functionality, and other details.
```

Comment:
This date is incorrect, as it was released on Tuesday, 5th February, 2019. Sorry I missed one of the versions in my dump 😢 

See: https://www.tiny.cloud/blog/tinymce-5-power-meets-beauty/

Reply:
Worth noting https://github.com/tinymce/tinymce-docs/blob/f19ee0db6b69d0647c6edc1994f727839b6d9c03/_includes/misc/supported_versions.md says the 4th February, 2019.

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1779/commits/bb176495886196c47aa96ca142dc929a0bd67fae